Spelling

bradyt opened the pull request:
I think sufix should be suffix.

As long as I was going to report it, I also checked in general with something like the following:

```
find . -name '*.dart' -print -exec ispell {} \;
```

I fumbled the keys halfway through the process, and finished it off in Emacs with something like the following:

```
M-x find-name-dired RET *.dart RET t Q sufix RET suffix RET Y
```

Similarly for `*.md` and `' +$' -> ''`.

I also checked `dart format . --fix` and `dart test`.

Let me know if you prefer prefixes, lowercase or one-liners in the commit title and body. And if you want an extra commit for the dart format change, or the two current commits squashed as one.

~~Oh, it's an API change, maybe I shouldn't have edited CHANGELOG.md.~~ I reverted the editing of `--exclude-sufix` in CHANGELOG.md and force pushed.

emanuel-braz commented:
Hi @bradyt , thank you for the contribution!

> I think sufix should be suffix.

Yes, you are correct!


>Similarly for *.md and ' +$' -> ''.

I can also add a finder using regular expressions.
e.g. `dlcov -c 80 -e **/*.g.dart` 
It's open to discussion, if you need this feature, let me know.

> I also checked dart format . --fix and dart test.

If you are using linux or mac os, there is a makefile script, so you can run `make analyze` in the project root folder.

>Let me know if you prefer prefixes, lowercase or one-liners in the commit title and body.

I follow this convention [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/).
In addition to this, I will add a PR template, too.

> Oh, it's an API change, maybe I shouldn't have edited CHANGELOG.md. I reverted the editing of --exclude-sufix in CHANGELOG.md and force pushed.

Don't worry about that, you can go this way, since we do not have [UNRELEASED] section in this changelog, so I will add this information before generate a new release.

bradyt commented:
>> Similarly for `*.md` and `' +$' -> ''`.
>
> I can also add a finder using regular expressions.
> e.g. dlcov -c 80 -e **/*.g.dart
> It's open to discussion, if you need this feature, let me know.

Oh, I was speaking in code, apologies. What I meant was that I removed trailing whitespace from markdown files in the second commit, with regex that matches `' +$'`, in other words, remove any positive number of spaces before end-of-line. Just a style change to your README.md.

>> Let me know if you prefer prefixes, lowercase or one-liners in the commit title and body.
>
> I follow this convention [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/).

Sounds like the commit messages should be written as follows:

```
style: remove trailing whitespace
fix!: change spelling of sufix to suffix, repostory to repository
```

I've rewritten the commit messages and force pushed.

emanuel-braz commented:
I was using double space before end-of-line, in order to break lines hahaha

I just tested this PR, let's merge it!
